File: Data Structure/String/Easy/LET_no520_Detect Capital.cpp
class Solution {
public:
    bool detectCapitalUse(string word) {
        int caps = 0;
        for (char c : word)
            caps += 'A' <= c && c <= 'Z';
        
        return caps == word.size() || caps == 0 || caps == 1 && 'A' <= word[0] && word[0] <= 'Z';
    }
};

File: Data Structure/String/Easy/LET_no520_Detect Capital.py
class Solution:
    def detectCapitalUse(self, word: str) -> bool:
        caps = 0
        for c in word:
            caps += 'A' <= c <= 'Z'
        return caps == len(word) or (caps == 1 and 'A' <= word[0] <= 'Z') or caps == 0